samples/cellular/coap_client/src/main.c

```diff
@@ -185,6 +185,18 @@ static void modem_configure(void)
 #endif /* defined(CONFIG_LTE_LINK_CONTROL) */
 }
 
+/**@brief Truns of LTE link and Shuts down modem to provide LTE link.
+ */
+static void modem_shutdown(void)
+{
+#if defined(CONFIG_LTE_LINK_CONTROL)
+	lte_lc_power_off();	
+#endif /* defined(CONFIG_LTE_LINK_CONTROL) */
```

Comment:
We should do a CFUN=0 here if the LTE LC is not enabled.

Reply:
nrf_modem_lib_shutdown() this does a cfun 0 after getting the status of device.